plugins/chat/spec/services/actions/publish_and_follow_direct_message_channel_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+RSpec.describe Chat::Action::PublishAndFollowDirectMessageChannel do
+  describe ".call" do
```

Comment:
Im wondering if we need this indentation level each time... It's very implicit and it's the only top level function we accept in a service, so maybe we just don't have it per convention in services?

Reply:
Indeed, since all these objects are method objects we could do what you suggest :slightly_smiling_face: